src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -1063,27 +1063,6 @@ public void Test()
             await InsertText(code, textToInsert, expectDocumentAnalysis: true);
         }
 
-        [Fact]
-        public void VBPropertyTest()
-        {
-            var markup = @"Class C
-    Default Public Property G(x As Integer) As Integer
-        Get
-            $$
-        End Get
-        Set(value As Integer)
-        End Set
-    End Property
-End Class";
-            MarkupTestFile.GetPosition(markup, out var code, out int position);
-
-            var root = Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.ParseCompilationUnit(code);
-            var property = root.FindToken(position).Parent.FirstAncestorOrSelf<Microsoft.CodeAnalysis.VisualBasic.Syntax.PropertyBlockSyntax>();
-            var memberId = Microsoft.CodeAnalysis.VisualBasic.LanguageServices.VisualBasicSyntaxFacts.Instance.GetMethodLevelMemberId(root, property);
-
-            Assert.Equal(0, memberId);
-        }
```

Comment:
removed test entirely.  all it does it test a low-level helper `GetMethodLevelMemberId` which isn't even something in teh new impl.  can't tell what the purpose of this test was.